ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsRuntimes.resource

```diff
@@ -28,12 +28,11 @@ Upload Serving Runtime Template
     [Documentation]    Uploads via UI a YAML file containing a custom Serving Runtime definition
     [Arguments]    ${runtime_filepath}    ${serving_platform}
     Click Add Serving Runtime Template Button
-    SeleniumLibrary.Click Element   ${UPLOAD_RUNTIME_BTN_XP}
+    Element Should Be Enabled   ${UPLOAD_RUNTIME_BTN_XP}
     ${rc}    ${pwd}=    Run And Return Rc And Output    echo $PWD
     Should Be Equal As Integers    ${rc}    ${0}
     SeleniumLibrary.Choose File    ${EDITOR_RUNTIME_BTN_XP}//input[@type="file"]    ${pwd}/${runtime_filepath}
-    Run Keyword And Continue On Failure    SeleniumLibrary.Wait Until Page Contains Element    //span[text()="kind"]
-    Run Keyword And Continue On Failure    SeleniumLibrary.Wait Until Page Contains Element    //span[text()="ServingRuntime"]
```

Comment:
I think we need these 2 checks. They were added to ensure the YAML was loaded in the UI after the file upload

Reply:
Restored those checks